vmaas/types.go

```diff
@@ -45,6 +45,17 @@ type Request struct {
 	UseCsaf       bool `json:"use_csaf"`
 }
 
+type CvesRequest struct {
+	CveIDs              []string `json:"cve_list"`
+	ModifiedSince       string   `json:"modified_since"`
+	PublishedSince      string   `json:"published_since"`
```

Comment:
would it make sense to change these to `*time.Time` type? That way we don't need to do the parsing here but in RedHatInsights/vmaas repo where request would need to be unmarshalled either way (with gin.context.BindJSON) and timestamp parsing can be done in Unmarshal func. BTW, we might need something like https://github.com/RedHatInsights/patchman-engine/blob/master/base/types/timestamp.go to correctly unmarshal all types of timestamps in the controller in RedHatInsights/vmaas